touchnet/models/qwen2_audio/inference_qwen2_audio.py

```
@@ -78,19 +92,69 @@
     for infos, audios in dataloader:
-        prompts = [QWEN2_AUDIO_TEMPLATE_FOR_S2T.replace("<|INSTRUCT|>", args.instruct)] * len(infos)
-        inputs = processor(
-            text=prompts,
-            audio=audios,
-            return_tensors="pt",
-            sampling_rate=processor.feature_extractor.sampling_rate,
-            padding=True
-        )
+        input_ids_buf, attention_mask_buf = [], []
+        input_features_buf, feature_attention_mask_buf = [], []
+        for i, audio in enumerate(audios):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

(B007)